Weights/include/CGAL/Weights/arap_weights.h

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2014 GeometryFactory SARL (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Yin Xu, Andreas Fabri and Ilker O. Yaz
+
+#ifndef CGAL_ARAP_WEIGHTS_H
+#define CGAL_ARAP_WEIGHTS_H
+
+#include "cotangent_weights.h"
+
+namespace CGAL {
+
+/// \ingroup PkgWeightsRefARAPWeights
```

Comment:
This line gives in the overnight documentation build the warning:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-6.1-Ic-22/include/CGAL/Weights/arap_weights.h:21: warning: Found non-existing group 'PkgWeightsRefARAPWeights' for the command '@ingroup', ignoring command
```

Either the group does not exist (no `\defgroup PkgWeightsRefARAPWeights` or the the name of the group is misspelled.

Reply:
I am also surprised to see an include statement with quotes instead of `<..>`

Reply:
Thank you, I didn't see it! It just moved to Surface_deformation, but it will be fixed tomorrow.